layouts/partials/head.html

```diff
@@ -89,10 +85,12 @@
 <meta property="og:type" content="{{ $ogType }}">
 
 {{ if .HasShortcode "mermaid" }}
-<!-- Copied from https://unpkg.com/mermaid@8.11.2/dist/mermaid.min.js -->
+<!-- Copied from https://unpkg.com/mermaid@8.13.4/dist/mermaid.min.js -->
 <script async src="{{ "js/mermaid.min.js" | relURL }}"></script>
 {{ end }}
 
+<script src="/js/jquery-3.6.0.min.js" intregrity="sha384-vtXRMe3mGCbOeY7l30aIg8H9p3GdeSe4IFlP6G8JMa7o7lXvnz3GFKzPxzJdPfGK" crossorigin="anonymous"></script>
```

Comment:
Not sure why this PR moved `jquery` to the lower place.

Reply:
@chrismetz09 was there a reason?

Reply:
early troubleshooting task exploring moving script files to end of file.